src/api/spec/support/cache.rb

```diff
@@ -0,0 +1,6 @@
+# reset the cache before each test
+RSpec.configure do |config|
+  config.before(:suite) do
```

Comment:
Before :suite executes this before running all tests, not before every test. If you want to execute it before every test you need ``config.before(:each) ...``

Reply:
that makes sense :)